window.go

```diff
@@ -64,6 +64,9 @@ type Window interface {
 	// SetOnClosed sets a function that runs when the window is closed.
 	SetOnClosed(func())
 
+	// SetCloseIntercept sets a function that runs instead of closing if defined
```

Comment:
Can you add a "." to this, and add another line instructing people to call `Close()` to confirm window closure?